packages/x-charts-headless-docs/app/page.tsx

```diff
@@ -0,0 +1,157 @@
+        two packages:
+      </p>
+      <ul>
+        <li>
+          <strong>@mui/x-charts-headless</strong>: Framework-agnostic core with inline styles and a
+          custom theme system
+        </li>
+        <li>
+          <strong>@mui/x-charts-material</strong>: Material UI integration layer using material
+          components and theming
+        </li>
+      </ul>
+
+      <div>
+        <h2>Headless Pie Chart</h2>
+        <PieChart.Root {...data}>
+          <PieChart.Surface>
+            <PieChart.Plot>{(item, index) => <PieChart.Arc key={index} {...item} />}</PieChart.Plot>
+            <PieChart.LabelPlot>
+              {(item, index) => <PieChart.ArcLabel key={index} {...item} />}
+            </PieChart.LabelPlot>
+          </PieChart.Surface>
+        </PieChart.Root>
+      </div>
+      <div>
+        <h2>Material UI Pie Chart</h2>
+        <PieChartMaterial {...data} />
+      </div>
+      <div>
+        <h2>Headless + CSS</h2>
+        <PieChart.Root {...data}>
+          <PieChart.Surface className="surface">
+            <PieChart.Plot>
+              {(item, index) => <PieChart.Arc key={index} className="arc" {...item} />}
+            </PieChart.Plot>
+            <PieChart.LabelPlot>
+              {(item, index) => <PieChart.ArcLabel key={index} className="arc-label" {...item} />}
+            </PieChart.LabelPlot>
+          </PieChart.Surface>
+          <PieTooltip />
+        </PieChart.Root>
+      </div>
+    </div>
+  );
+}
+
+function PieTooltip() {
+  const tooltipData = useItemTooltip();
+  const svgRef = useSvgRef();
+  const [position, setPosition] = React.useState<{ x: number; y: number } | null>(null);
+  const drawingArea = useDrawingArea();
+
+  React.useEffect(() => {
+    const svgElement = svgRef.current;
+    if (!svgElement) {
+      return undefined;
+    }
+
+    function handleMove(event: MouseEvent) {
+      if (!svgElement) {
+        return;
+      }
+
+      setPosition({
+        x: event.clientX,
+        y: event.clientY,
+      });
+    }
+
+    function handleLeave() {
+      setPosition(null);
+    }
+
+    svgElement.addEventListener('pointermove', handleMove);
+    svgElement.addEventListener('pointerleave', handleLeave);
+    return () => {
+      svgElement.removeEventListener('pointermove', handleMove);
+      svgElement.removeEventListener('pointerleave', handleLeave);
+    };
+  }, [svgRef, drawingArea]);
+
+  return position && tooltipData ? (
+    <div
+      className="tooltip-container"
+      style={{
+        position: 'absolute',
+        top: position.y,
+        left: position.x,
+      }}
+      data-index={tooltipData.identifier.dataIndex}
+      data-series={tooltipData.identifier.seriesId}
+    >
+      <div className="tooltip-color-box"></div>
+      <div className="tooltip-data">
+        {tooltipData.label}: {tooltipData.formattedValue}
+      </div>
+    </div>
+  ) : null;
+}
```

Comment:
Should look like similar to this in the final implementation

```suggestion
function PieTooltip() {
  const {data, position} = useItemTooltip();
  
  if (!data || !position) return null;

  return (
    <Popper
      className="tooltip-container"
      popperPositionProp={{ x: position.x, y: position.y }}
    >
      <div className="tooltip-color-box"></div>
      <div className="tooltip-data">
        {data.label}: {data.formattedValue}
      </div>
    </div>
  );
}
```